docs/Configuration/yaml-files.md

```diff
@@ -261,6 +261,8 @@ controls:
     enable: true
     mode: voluntary
     webhook_url: https://example.org/webhook_handler
+  windows_migration: # Available in Fleet Premium
+    enable: true
```

Comment:
Hey @rachaelshaw, is there a good reason to create a nested object here? If not, maybe we pull this out (de-nest) and call it `windows_mdm_migration_enabled`? Easier to rename/change later.

I think the config to turn on Windows MDM (equivalent) to our toggle/switch in the UI is `windows_enabled_and_configured`: https://github.com/fleetdm/fleet/pull/22398/files#diff-b71104232d8fbaaf4bd537065533cbf39ec5f9bf028ec1d8083346953ed5178dR237

I think it makes sense to move this new config to sit alongside the existing one. What do you think?

Reply:
I think the same nesting v. de-nesting question to your proposed API changes as well.

Reply:
The nested object was mostly to match the format of the macOS migration settings, but also to leave room in case we add any other migration settings on this page in the future. Do you foresee that being a possibility?

If not, we could change to `windows_mdm_migration_enabled` and then maybe deprecate that key in favor of a `windows_migration` object if we ever add additional settings.